.github/workflows/runtests.yml

```
@@ -66,8 +66,32 @@ jobs:
+          environment-file: './.github/doc_environment.yml'
+
+      - uses: ./.github/actions/build-docs
 
+      - name: publish docs to netlify
+        shell: bash -l {0}
+        env:
+          QUARTO_PRINT_STACK: true
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+        run: |
+            quarto publish docs --no-prompt --no-browser
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)